Description: Another spuriously failing test
 The symptom is the same as in 03_spurious_test_failure, the cause is a race triggered by recent python 2.7 versions when client and server share a socket in the same process.
 .
Author: Vincent Ladeuil <v.ladeuil+lp@free.fr>

---

Index: unstable/bzrlib/tests/test_smart_transport.py
===================================================================
--- unstable.orig/bzrlib/tests/test_smart_transport.py
+++ unstable/bzrlib/tests/test_smart_transport.py
@@ -1462,7 +1462,6 @@ class TestSmartTCPServer(tests.TestCase)
         self.assertThat(log, DocTestMatches("""\
     INFO  Requested to stop gracefully
 ... Stopping SmartServerSocketStreamMedium(client=('127.0.0.1', ...
-    INFO  Waiting for 1 client(s) to finish
 """, flags=doctest.ELLIPSIS|doctest.REPORT_UDIFF))
 
     def test_stop_gracefully_tells_handlers_to_stop(self):
